apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -833,7 +832,7 @@ type BareMetalHostStatus struct {
 	// The last error message reported by the provisioning subsystem.
 	ErrorMessage string `json:"errorMessage"`
 
-	// Whether or not the host is currently powered on. This field may get
+	// The currently detected power state of the host. This field may get
 	// briefly out of sync with the actual state of the hardware while
 	// provisioning processes are running.
```

Comment:
I meant we could say 'currently detected' _instead_ of all the stuff about being briefly out of sync.
I feel like it goes without saying that we are in a distributed system and detection is not instananeous.

Reply:
I'd rather keep the second part here because it's a common source of confusion.